src/Sylius/Bundle/AdminBundle/spec/Provider/StatisticsDataProviderSpec.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace spec\Sylius\Bundle\AdminBundle\Provider;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\AdminBundle\Provider\StatisticsDataProvider;
+use Sylius\Bundle\MoneyBundle\Formatter\MoneyFormatterInterface;
+use Sylius\Component\Core\Dashboard\DashboardStatistics;
+use Sylius\Component\Core\Dashboard\DashboardStatisticsProviderInterface;
+use Sylius\Component\Core\Dashboard\Interval;
+use Sylius\Component\Core\Dashboard\SalesDataProviderInterface;
+use Sylius\Component\Core\Dashboard\SalesSummaryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+
+
+class StatisticsDataProviderSpec extends ObjectBehavior
+{
+    function let(DashboardStatisticsProviderInterface $statisticsProvider, SalesDataProviderInterface $salesDataProvider, MoneyFormatterInterface $moneyFormatter): void
+    {
+        $this->beConstructedWith($statisticsProvider, $salesDataProvider, $moneyFormatter);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(StatisticsDataProvider::class);
+    }
```

Comment:
you can omit this test but you should add test to check if this class implements the proper interface 
```suggestion
    function it_implements_a_statistics_data_provider_interface(): void
    {
        $this->shouldImplement(StatisticsDataProviderInterface::class);
    }
```